src/SLCore/Listener/FixSuggestion/Models/FileEditDto.cs

```diff
@@ -0,0 +1,25 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarLint.VisualStudio.SLCore.Common.Models;
+
+namespace SonarLint.VisualStudio.SLCore.Listener.FixSuggestion.Models;
+
+public record FileEditDto(FileUri idePath, List<ChangesDto> changes);
```

Comment:
I don't think FileUri is serializable. It should probably be string.

Reply:
There is another FileEditDto which in the Java side had it as a URI. However, in this FileEditDto, the java side is a Path. Makes sense to change it to string 😉